db/log_reader.cc

```diff
@@ -329,6 +356,54 @@ bool Reader::ReadRecord(Slice* record, std::string* scratch,
   return false;
 }
 
+void Reader::MaybeVerifyPredecessorWALInfo(
+    WALRecoveryMode wal_recovery_mode, Slice fragment,
+    const PredecessorWALInfo& expected_predecessor_wal_info) {
```

Comment:
"expected" is confusing to me, because both what is observed in DB recovery and what is saved in the special WAL entry are "expected" to match the other. Similarly, `predecessor_wal_info_` is ambiguous with no comment.

How about `observed_predecessor_wal_info_` and `recorded_predecessor_wal_info`?

Reply:
Fixed